hapi-fhir-docs/src/main/resources/ca/uhn/hapi/fhir/changelog/6_2_0/4271-migration-failure-with-oracle-19c-as-database-support.yaml

```
@@ -0,0 +1,4 @@
+---
+type: fix
+issue: 4271
+title: "Database migration steps are failing due to database engine not being able to drop non existent indexes with Oracle 19C."
```

minor: This reads more like a summary of the previous problem than a description of what has changed.

```suggestion
title: "Database migration steps were failing with Oracle 19C. This has been fixed by allowing the database engine to skip dropping non-existent indexes."
```

---

thanks for the suggestion, will incorporate.